php/Dockerfile

```diff
@@ -2,6 +2,23 @@ FROM debian:jessie
 MAINTAINER Damien Carcel <damien.carcel@gmail.com>
 
 ENV DEBIAN_FRONTEND noninteractive
+ENV GOSU_VERSION 1.9
+
+# Install user impersonator
+RUN set -x \
+    && apt-get update && apt-get install -y --no-install-recommends ca-certificates wget && rm -rf /var/lib/apt/lists/* \
+    && dpkgArch="$(dpkg --print-architecture | awk -F- '{ print $NF }')" \
+    && wget -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch" \
```

Comment:
Use directly version and architecture (next line too) rather than env variables.